config/helm/kustomization.yaml

```diff
@@ -105,12 +107,6 @@ patches:
     patch: |-
       - op: remove
         path: /spec/template/spec/securityContext/seccompProfile
-  - target:
-      kind: CustomResourceDefinition
-      name: (ipaddressclaims\.ipam|extensionconfigs\.runtime|ipaddresses\.ipam).cluster.x-k8s.io
-    patch: |-
-      - op: remove
-        path: /metadata/creationTimestamp
```

Comment:
is this intended? I remember something breaking without this patch.

Reply:
Yeah, this is fixed in upstream. Keeping it isn't possible since otherwise kustomize would complain about this field missing (you can't say "idempotently delete this and ignore if not exists").